src/cli/cli_commissioner.cpp

```diff
@@ -192,6 +289,25 @@ template <> otError Commissioner::Process<Cmd("joiner")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli commissioner mgmtget
+ * @code
+ * commissioner mgmtget locator sessionid
+ * Done
+ * @endcode
+ * @cparam commissioner mgmtget [locator] [sessionid] <!--
+ * -->                          [steeringdata] [joinerudpport] <!--
+ * -->                          [-x @ca{TLVs}]
+ *   * `locator`: Border Router RLOC16.
+ *   * `sessionid`: Session ID of the %Commissioner.
+ *   * `steeringdata`: Steering data.
+ *   * `joinerudpport`: UDP port of the %Joiner.
```

Comment:
"Joiner UDP port" instead?
Mainly suggestion of removing "the" in "the %Joiner" since there can be more than one node acting as `Joiner`